views/shop/refundtopoints.tt.text

```diff
@@ -9,6 +9,8 @@
 
 .complete=Your exchange has been completed.
 
+.nextrefund=You won't be able to use this tool again until after [[date]].
```

Comment:
Yeah, that was just totally off the cuff, people should always feel free to rewrite anything I specify!

"You won't be able to refund paid time to points again until [[date]]"?